bindings/ergo-lib-wasm/src/json.rs

```diff
@@ -37,6 +37,8 @@ pub(crate) struct TransactionJsonEip12 {
 
 impl From<Transaction> for TransactionJsonEip12 {
     fn from(t: Transaction) -> Self {
+        // Following unwraps are fine since we're converting from BoundedVec to Vec.
```

Comment:
I'm not sure why try_into() is used here. It's an infallible op. There is as_vec() and From implemented for getting the Vec out of BoundedVec.

Reply:
Thanks, making the change now!